### ISSUE TITLE
Publish snapshots

### ISSUE THREAD

**Issue by jberkel** [sweble/sweble-wikitext#85]
The old snapshot hosts (http://mojo-maven.cs.fau.de/content/repositories/public) are no longer accessible. Perhaps [Github Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages) could be used to publish snapshots after successful builds.

**Comment by wetneb** [comment]
Or publishing to the Maven Central snapshot repository, too.

**Comment by jberkel** [comment]
> Or publishing to the Maven Central snapshot repository, too.

oh, I thought you couldn't upload snapshots to central…

**Comment by wetneb** [comment]
Yes it is a separate repository, you can use the URL `https://oss.sonatype.org/content/repositories/snapshots` for that :)

**Comment by jberkel** [comment]
OK, it's been a while since I last deployed to sonatype. We need to register the project with them first (via JIRA ticket). And for this it looks like we would need DNS access to sweble.org, to prove ownership. 

https://central.sonatype.org/faq/verify-ownership/

**Comment by wetneb** [comment]
Yes let's set up some website at `sweble.github.io` and ask @hannesd to point the domain to it.

Or we can publish it under the `io.github.sweble` namespace

**Comment by jberkel** [comment]
Any ideas on how to generate the website? A static site generator? Use maven-site? (gasp)

**Comment by wetneb** [comment]
I have never used maven-site. I could set up Jekyll, which works quite well, but happy with any option you are familiar with.

**Comment by jberkel** [comment]
We could use maven-site to generate a standard site (with javadocs, stats etc.), and link to it from the main page. Jekyll is fine.

First step would be to create a landing page, listing the current version + maven coordinates and a short intro.

**Comment by wetneb** [comment]
I don't seem to have the rights to invite you to the org but it looks like I can give you full rights on the website repo, so I have sent you an invite for that. The floor is all yours!
